plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -325,7 +325,24 @@
         <mavenCoordinates
               artifactId="google-cloud-translate"
               groupId="com.google.cloud"
-              version="1.3.1" />
+              version="LATEST" />
+      </libraryFile>
+    </library>
+
+    <library
+          id="cloudlogging"
+          group="clientapis"
+          name="%stackdriverlogging.name"
+          tooltip="%stackdriverlogging.tooltip"
+          javaVersion="1.8"
+          siteUri="https://cloud.google.com/translate/docs/reference/libraries" >
+      <libraryDependency id="googlecloudcore" />
```

Comment:
I think the latter group is going to be a req for any library that uses the GRPC transport.